user_doc/db_recovery.md

```diff
@@ -55,7 +109,32 @@ If the Packages file now passes the verify step, then as an additional sanity ch
 If this query passes without generating any messages to standard error, then it is time to rebuild the indexes by invoking:
 
 ```
-# rpm -v --rebuilddb
+# rpmdb -v --rebuilddb
+```
+
+At this point you should have a functioning RPM database again.
+
+
+# If all fails
+
+
+If the recovery failed, then a bug should be reported. Note that this is a rare event and it is possible that your system took more damage than just the rpm database.
+
+When creating the bug report, provide the tar.gz backup of the RPM DB as an attachment. If available also provide the list of installed packages. First try
+
+```
+# rpm -qa 1> /dev/null
```

Comment:
That should be `2>`, no? The above will only output  warning/error messages, not packages.